pytorch_lightning/loggers/base.py

```diff
@@ -310,7 +311,16 @@ def finalize(self, status: str) -> None:
         self.save()
 
     def close(self) -> None:
-        """Do any cleanup that is necessary to close an experiment."""
+        """
+        .. deprecated:: v1.5
+          This method was deprecated in v1.5 and will be removed in v1.7
+          in favor of `LightningLoggerBase.save`
+
+        """
```

Comment:
```suggestion
        """
        .. deprecated:: v1.5
            This method was deprecated in v1.5 and will be removed in v1.7
            in favor of `LightningLoggerBase.save`
        """
```
also, generally we would keep the documentation in place during the deprecation phase